lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -107,6 +108,10 @@ static Optional<APSInt> getExtendedConstant(Value operand, Attribute constant,
   if (destWidth < 0)
     return {};
 
+  // InvalidValue inputs simply read as zero.
+  if (auto result = constant.dyn_cast_or_null<InvalidValueAttr>())
+    return APSInt(destWidth, operand.getType().cast<IntType>().isUnsigned());
+
```

Comment:
Can you update the the comment on this function to reflect this change in behavior?

Reply:
Done :+1: